packages/smart-wallet/test/swingsetTests/upgradeWalletFactory/walletFactory-V2.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @file fixture for an upgrade of the primary walletFactory contract,
+ * packages/smart-wallet/src/walletFactory.js
+ *
+ * This variant has minor changes to the returned strings that make it
+ * identifiable, to demonstrate that upgrade has occurred.
+ */
+
+import { M, makeExo, mustMatch } from '@agoric/store';
+import { makeAtomicProvider } from '@agoric/store/src/stores/store-utils.js';
+import { makeScalarBigMapStore } from '@agoric/vat-data';
+import { E } from '@endo/far';
+import { prepareSmartWallet } from '../../../src/smartWallet.js';
+import { shape } from '../../../src/typeGuards.js';
+import {
+  makeAssetRegistry,
+  publishDepositFacet,
+} from '../../../src/walletFactory.js';
+
+/**
+ *
+ * @type {typeof import('../../../src/walletFactory.js').start}
+ */
+export const start = async (zcf, privateArgs, baggage) => {
+  const { agoricNames, board, assetPublisher } = zcf.getTerms();
+
+  const zoe = zcf.getZoeService();
+  const { storageNode, walletBridgeManager } = privateArgs;
+
+  /** @type {MapStore<string, import('../../../src/smartWallet').SmartWallet>} */
+  const walletsByAddress = makeScalarBigMapStore('walletsByAddress');
+  const provider = makeAtomicProvider(walletsByAddress);
+
+  const handleWalletAction = makeExo(
+    'walletActionHandler',
+    M.interface('walletActionHandlerI', {
+      fromBridge: M.call(shape.WalletBridgeMsg).returns(M.promise()),
+    }),
+    {
+      /**
+       *
+       * @param {import('../../../src/types.js').WalletBridgeMsg} obj validated by shape.WalletBridgeMsg
+       */
+      fromBridge: async obj => {
+        console.log('walletFactory.fromBridge:', obj);
```

Comment:
I like to make at least one thing detectably different in the upgraded contract so I can tell that I'm not fooling myself and re-running the original.